net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Why not 7? Just update `gnome2_icon_cache_update` to use `xdg_icon_cache_update` from xdg-utils instead. 7 should work here.

Reply:
Will try, seems straightforward.

Reply:
Rebased and updated to EAPI 7 and xdg-utils.